src/Discovery/Discovery.tsx

```diff
@@ -277,19 +287,25 @@ const Discovery: React.FunctionComponent<Props> = (props: Props) => {
     // in the table and the study description.
     // ---
     const searchableFields = config.features.search.searchBar.searchableTextFields;
-    if (searchableFields) {
+    if (searchableFields && 0==1) {
```

Comment:
we need to ass this back, also the code needs to be able to handle nested fields in these `searchableFields`